example/k3d/scripts/create.bash

```diff
@@ -4,4 +4,5 @@ k3d create \
   --name agent-k3d \
   --publish 30080:30080 \
   --api-port 50443 \
-  -v /var/lib/k3d/agent-k3d/storage/:/var/lib/rancher/k3s/storage/
+  -v /var/lib/k3d/agent-k3d/storage/:/var/lib/rancher/k3s/storage/ \
+  -v /etc/machine-id:/etc/machine-id
```

Comment:
This is needed for the Loki to work, where /etc/machine-id is bind-mounted. Promtail didn't need it unless you were reading from the systemd journal, but the Agent config always adds it.